ddhcpd/files/etc/init.d/ddhcpd

```diff
@@ -43,7 +69,15 @@ start_service() {
   procd_append_param command -i "$server_interface"
   procd_append_param command -N "$block_network"
   procd_append_param command -H "/etc/ddhcpd.hook"
-  config_foreach dhcp_option dhcp_option 
+
+  [ "$use_config_dhcp" == "1" ] && {
```

Comment:
Remove this line, unindent the following

Reply:
That would enable the behavior per default, is that desired?

Reply:
Yes. The lease time value in the configuration file is unused until now. When someone is defining that value by hand via dhcp_option config blocks, this will overwrite the value set by your code.
So there is no conflict.

Reply:
Okay, changed. It first loads all the settings from `/etc/config/dhcp` and then load `ddhcpd`